easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.1-2023a.yml

```diff
@@ -22,3 +22,7 @@ easyconfigs:
         from-pr: 20784
   - Valgrind-3.21.0-gompi-2023a.eb
   - OrthoFinder-2.5.5-foss-2023a.eb
+  - OpenFOAM-v2312-foss-2023a.eb:
```

Comment:
@bedroge Let's start using EasyBuild v4.9.2 for this?

Reply:
Sure!